packages/flame/test/components/component_test.dart

```diff
@@ -200,6 +200,56 @@ void main() {
       );
     });
   });
+
+  group('Component descendants', () {
+    flameGame.test(
```

Comment:
Please prefer `testWithFlameGame()` (see #1358 for reasons), which has exactly the same signature. We're slowly refactoring all test code to stop using `flameGame.test()`.

Reply:
done